instrumentation/opentelemetry_ecto/lib/opentelemetry_ecto.ex

```diff
@@ -36,6 +36,9 @@ defmodule OpentelemetryEcto do
       defaults to the concatenation of the event name with periods, e.g.
       `"blog.repo.query"`. This will always be followed with a colon and the
       source (the table name for SQL adapters).
+    * `:attributes` - additional attributes to include in the span. If there
```

Comment:
Let's call this additional_attributes. I don't want to overload that word as an option when it comes to optional attributes in the future